lib/Basics/ConditionVariable.h

```diff
@@ -99,10 +101,11 @@ class ConditionVariable {
 
   //////////////////////////////////////////////////////////////////////////////
   /// @brief waits for an event with timeout in micro seconds
-  /// returns true when the condition was signaled, false on timeout 
+  /// returns true when the condition was signaled, false on timeout
   //////////////////////////////////////////////////////////////////////////////
 
   bool wait(uint64_t);
+  bool wait(std::chrono::microseconds);
```

Comment:
If your're adding this, maybe change this line to use `std::chrono` directly too: https://github.com/arangodb/arangodb/blob/ed94894103c171267b8dce4a9fb578afe918c46e/lib/Basics/ConditionLocker.cpp#L86

Reply:
@neunhoef Looking to you for merge.

Reply:
OK. But keep in mind that duration</*signed integer type of at least 55 bits*/, std::micro> so 55bits only. if you need 64 bits, do not use microseconds.